### ISSUE TITLE
sbt.jse.command is not used for AutoDetect engine.

### ISSUE THREAD

**Issue by margussipria** [sbt/sbt-js-engine#59]
`sbt.jse.command` is not used for `AutoDetect` engine.

you need to add both `sbt.jse.engineType` and `sbt.jse.command`

```scala
export SBT_OPTS="$SBT_OPTS -Dsbt.jse.engineType=Node -Dsbt.jse.command=/usr/local/bin/node"
```

**Comment by margussipria** [comment]
I think I left out most relevant part... it does not work IF there is no node in PATH... problem only happened on our old CI system with multiple node versions available, but non of them was in PATH.

`autoDetectNode` requires node to be in path... it could try to run binary in `sbt.jse.command` with `--version` argument, but I don't know if that would be desirable.

**Comment by margussipria** [comment]
I think line 
```
val nodeExists = Try(Process("node --version").!!).isSuccess
```
should be something like this:
```
val nodeExists = Try(Process(s"${LocalEngine.path(command, "node")} --version").!!).isSuccess // didn't test actual code
```
path method is used for really running node 
```
val args = immutable.Seq(path(command, "node")) ++ stdArgs
```
but quite many binaries do support `--version` argument, so it also detect name of binary...

**Comment by mkurz** [comment]
I think it's safe to expect the `--version` flag to be available.
Also your suggested fix makes sense, I implemented it in #97 
This will ship in version 1.3.0.
Thanks!

**Comment by mkurz** [comment]
Fixed by #97
